dds/DCPS/RTPS/RtpsDiscovery.cpp

```diff
@@ -508,10 +510,22 @@ RtpsDiscovery::Config::discovery_config(ACE_Configuration_Heap& cf)
             config->max_spdp_sequence_msg_reset_check(value);
           } else {
             ACE_ERROR_RETURN((LM_ERROR,
-              ACE_TEXT("(%P|%t) RtpsDiscovery::Config::discovery_config(): ")
-              ACE_TEXT("Invalid entry (%C) for MaxSpdpSequenceMsgResetChecks in ")
-              ACE_TEXT("[rtps_discovery/%C] section.\n"),
-              string_value.c_str(), rtps_name.c_str()), -1);
+                              ACE_TEXT("(%P|%t) RtpsDiscovery::Config::discovery_config(): ")
+                              ACE_TEXT("Invalid entry (%C) for MaxSpdpSequenceMsgResetChecks in ")
+                              ACE_TEXT("[rtps_discovery/%C] section.\n"),
+                              string_value.c_str(), rtps_name.c_str()), -1);
+          }
+        } else if (name == "UdpMaxMessageSize") {
```

Comment:
```suggestion
        } else if (name == "SedpMaxMessageSize") {
```
making it clear that this doesn't impact SPDP.  The use of "UDP" for the constant in TransportSendStrategy was because it's UDP's inherent max size, which we can't change.